src/classes/i18n.js

```diff
@@ -211,10 +211,10 @@ export default class i18n {
                     retry_count = MAX_RETRY;
                 }
 
-                async_cb(retry_count);
+                return retry_count;
             },
             (async_cb_res, next_cb) => next_cb(async_cb_res < MAX_RETRY),
-            err => cb(err ? err : null)
+            err => err ? err : null
```

Comment:
No need for callback or err handling, right? Since doWhilst would return a promise that'll handle the error?

Or perhaps since we're already using async/await, this can be done through a simple javascript for loop?

Reply:
> Or perhaps since we're already using async/await, this can be done through a simple javascript for loop?

I'll re-write this in for loop. Sec.

Reply:
Converted to for-loop https://github.com/anyTV/anytv-i18n/pull/10/commits/de3c765f5129eacae7d132dd17e98c14d399d0cf